### PR TITLE
Add monitorTypeSurfaces explainer

### PR DESCRIPTION
**Pull request by beaufortfrancois**
@eladalon1983 Please review

### CODE REVIEW

File: monitorTypeSurfaces_Explainer.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+```js
+// Hint the browser to exclude monitors from the choices offered to the user.
+const stream = await navigator.mediaDevices.getDisplayMedia({
+  monitorTypeSurfaces: "exclude",
+});
+
+// We still need to check though...
+const { displaySurface } = stream.getTracks()[0].getSettings();
+if (displaySurface == "monitor") {
+  // User has picked a monitor...
```
Comment (eladalon1983) [inline monitorTypeSurfaces_Explainer.md]: Could the comment more clearly indicate that the app sees this as an error? Maybe an exception could be raised? Wdyt?

File: monitorTypeSurfaces_Explainer.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+}
+```
```
Comment (eladalon1983) [inline monitorTypeSurfaces_Explainer.md:23]: Wdyt of adding a "Result (Chrome)" section and image, as in [this other explainer](https://github.com/eladalon1983/screen-share-explainers/blob/main/systemAudio_Explainer.md#result-chrome)?

**Review by eladalon1983 — APPROVED** [review]
Looks awesome. Thanks a lot!

File: monitorTypeSurfaces_Explainer.md (hunk 1 above)
Comment (beaufortfrancois) [reply monitorTypeSurfaces_Explainer.md]: Throwing an error looks good!

File: monitorTypeSurfaces_Explainer.md (hunk 2 above)
Comment (beaufortfrancois) [reply monitorTypeSurfaces_Explainer.md:23]: I'm not sure how GitHub will handle URLs uploaded from my repo. Shall I share offline images you can upload later once the PR is merged?

```
## Result (Chrome)
![image1](https://github.com/eladalon1983/screen-share-explainers/assets/22117736/c4c3835a-254b-44d8-b9ca-c719a95ec1dd)
![image2](https://github.com/eladalon1983/screen-share-explainers/assets/22117736/4e7c4580-3113-4f53-93b5-955daa293ad9)
```

Comment (eladalon1983) [reply monitorTypeSurfaces_Explainer.md:23]: Works for me.

**Pull request merged**